web/.happo.js

```diff
@@ -0,0 +1,20 @@
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+
+  apiKey: process.env.HAPPO_API_KEY,
+  apiSecret: process.env.HAPPO_API_SECRET,
+
+  pages: [
+    { url: process.env.URL, title: 'Pull Request' }
```

Comment:
I might be wrong, but don't we want a comparison when we push to the main as well?

Reply:
The app when the PR is approved and the app when it is pushed to main would be the same so I don't think it is necessary. The title of "pull request" is there because when the url's are compared in Happo, the images with the same titles are compared.

Reply:
I see. Thank you for explaining this!